salesforce/10.22/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -54,7 +54,7 @@ The connector contains an in-memory structure that stores the replay IDs of even
 This in-memory structure uses the username in the connector configuration to persist into the object store to build a specific key. A redundancy mechanism avoids processing duplicate events by storing the same data under a backup key.
 Each persist operation consists of deleting the existing key and storing the new value, and if storing the new value fails, the connector retries, thus consuming 2 or more TPS per key.
 
-If you have multiple workers deployed in CloudHub, the *Replay topic listener* and *Replay channel listener* sources process each message on all Mule instances. To avoid this situation, you can deploy your application on a single worker and then use a shared queue to pass the messages to a different worker to process the message.
+If you have multiple workers deployed in CloudHub 1.0, the *Replay topic listener* and *Replay channel listener* sources process each message on all Mule instances. To avoid this situation, you can deploy your application on a single worker and then use a shared queue to pass the messages to a different worker to process the message.
```

Comment:
```suggestion
If you have multiple workers deployed in CloudHub, the *Replay topic listener* and *Replay channel listener* sources process each message on all Mule instances. To avoid this situation, you can deploy your application on a single worker and then use a shared queue to pass the messages to a different worker to process the message.
```